packages/graph/src/graph2/graph.ts

```diff
@@ -209,6 +200,13 @@ export class Graph2 extends SVGZoomWidget {
         this.zoomToFitLimit(1);
     }
 
+    resolveAnnoEventNode(): undefined | IVertex4Annotation {
+        const d3evt = d3Event();
+        const eventPath = d3evt?.sourceEvent?.path ?? d3evt.path;
```

Comment:
Inconsistent use of `d3evt?.` , should be `d3evt?.sourceEvent?.path ?? d3evt?.path`